README.md

```diff
@@ -69,15 +69,15 @@ configuration excludes `app.mod2.MyClass`
 
 * `APPMAP_CONFIG` specifies the configuration file to use. Defaults to `appmap.yml` in the
   current directory
-  
+
 * `APPMAP_LOG_LEVEL` specifies log level to use, from the set `CRITICAL`, `ERROR`,
   `WARNING`, `INFO`, `DEBUG`. Not case-sensitive, defaults to `WARNING`.
-  
+
 * `APPMAP_OUTPUT_DIR` specifies the root directory for writing AppMaps. Defaults to
   `tmp/appmap`.
-  
+
 ## Test Frameworks
-`appmap-python` supports recording [pytest](https://pytest.org) and `unittest`[coming soon]
+`appmap-python` supports recording [pytest](https://pytest.org) and `unittest`
```

Comment:
Even though it's in the standard library, it's probably worth putting a link to the unittest doc here. I'd say linking to the 3.9 page is fine.